<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="button1">
      <property name="minimumSize">
       <size>
        <width>202</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Thread #1</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="button2">
      <property name="minimumSize">
       <size>
        <width>202</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Thread #3</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>202</width>
        <height>38</height>
       </size>
      </property>
      <property name="text">
       <string>Integers generated by Thread #1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>202</width>
        <height>38</height>
       </size>
      </property>
      <property name="text">
       <string>Current content of the queue</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>202</width>
        <height>38</height>
       </size>
      </property>
      <property name="text">
       <string>Integers that are popped from the queue by thread #3</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QListWidget" name="listWidget1">
      <property name="minimumSize">
       <size>
        <width>202</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QListWidget" name="listWidget2">
      <property name="minimumSize">
       <size>
        <width>202</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QListWidget" name="listWidget3">
      <property name="minimumSize">
       <size>
        <width>202</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
